ui/shared/status/buttons/StatusIconTabButton.qml

```diff
@@ -0,0 +1,81 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtGraphicalEffects 1.13
+import "../../../imports"
+import "../../"
+import "../core"
+import "../"
+
+TabButton {
+    id: control
+
+    property color iconColor: Style.current.secondaryText
+    property color disabledColor: iconColor
+    property string name: ""
+
+    implicitWidth: 40
+    implicitHeight: 40
+
+    icon.height: 24
+    icon.width: 24
+    icon.color: {
+        if (!enabled) {
+            return control.disabledColor
+        }
+        return (hovered || checked) ? Style.current.blue : control.iconColor
```

Comment:
Also here: We can't use `blue` because the actual color in the designs have transparency. This needs an overhaul